extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -314,6 +319,12 @@ const actions = (state, action, data) => {
         ...state,
         [StateProperty.STORY_HAS_BACKGROUND_AUDIO_STATE]: !!data,
       });
+    // Shows or hides the play/pause controls.
+    case Action.TOGGLE_STORY_HAS_PLAYABLE:
```

Comment:
Sorry, why do we need a story-level and a page-level check? Couldn't we infer the story-level state given all of the page-level states?

Reply:
Just like there is a page and story level audio controls, there needs to be a page and story level play/pause control. Reason is, if the story doesn't have anything that can be paused, then the icon will not even show. If the story has sth "playable" but not on the current page, the icon will be disabled/translucent.

This approach was taken over the approach of adding and removing the icon per-page to avoid UI jumps, since the paused state only applies in the page level, as switching to another page automatically resumes the story.